dom/css.ts

```diff
@@ -109,7 +109,7 @@ function getComputedStyles (element : Element, pseudoElement : string|null = nul
 /**
  * Returns the CSS property value for the given property and element
  */
-export function getStyle (element : HTMLElement, property : string, pseudoElement : string|null = null) : string|number|null
+export function getStyle (element : HTMLElement|Window, property : string, pseudoElement : string|null = null) : string|number|null
 {
     let castedElement = element as mojave.types.StringIndexedHTMLElement;
```

Comment:
I think we need to extend `mojave.types.StringIndexedHTMLElement` to extend from `Window` as well.

Reply:
Nope, as soon as you are casting, only the new type is used from this point on. And the casting itself is not checked.